CodeStarterCamp_Week2/main.swift

```diff
@@ -6,3 +6,78 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
+import Foundation
+
+let lotto: Set<Int> = getlotto()
+
+func getlotto() -> Set<Int>{
+    
+    var lotto: Set<Int> = Set<Int>()
```

Comment:
```suggestion
    var lotto: Set<Int> = []
```

보통 빈 컬렉션 변수를 정의할 때는 대괄호를 사용합니다